internal/bminventory/inventory.go

```diff
@@ -2862,6 +2872,29 @@ func (b *bareMetalInventory) handleReplyError(params installer.PostStepReplyPara
 	return nil
 }
 
+func (b *bareMetalInventory) handleMediaDisconnection(params installer.PostStepReplyParams, ctx context.Context, log logrus.FieldLogger, h *models.Host) error {
+	statusInfo := fmt.Sprintf("%s - %s", string(models.HostStageFailed),
+		"Cannot read from the media (ISO) - media was likely disconnected")
+
+	// Install command reports its status with a different API, directly from the assisted-installer.
+	// Just adding our diagnose to the existing error message.
+	if swag.StringValue(h.Status) == models.HostStatusError && h.StatusInfo != nil {
+		// Add the message only once
+		if strings.Contains(*h.StatusInfo, statusInfo) {
+			return nil
+		}
+
+		statusInfo = fmt.Sprintf("%s\n%s", statusInfo, *h.StatusInfo)
+	} else if params.Reply.Error != "" {
+		statusInfo = fmt.Sprintf("%s\n%s", statusInfo, params.Reply.Error)
```

Comment:
I'm not sure we want to have \n in our statusInfo
@avishayt ?
Does UI support it? @jtomasek

Reply:
Already discussing it here https://github.com/openshift/assisted-service/pull/1867#discussion_r642854357

Reply:
yes, I tested it

Reply:
No please don't use \n